src/actions/VaccineActions.js

```diff
@@ -48,23 +52,27 @@ const withPermissions = async (dispatch, getState, func) => {
     return null;
   }
 
-  return func(dispatch, state);
+  return func(dispatch, getState);
```

Comment:
passing `getState` as just `state` gets stale in the scan call closure

Reply:
yeah, I wondered about that. thanks - over eager optimisation on my part that one